source/proof_resolution_mock/lib/state_id_mock_client.rb

```diff
@@ -0,0 +1,49 @@
+require 'proofer'
+
+module IdentityIdpFunctions
+  class StateIdMockClient < Proofer::Base
+    vendor_name 'StateIdMock'
+
+    required_attributes :state_id_number, :state_id_type, :state_id_jurisdiction
+
+    optional_attributes :uuid, :uuid_prefix
+
+    stage :state_id
+
+    SUPPORTED_STATES = %w[
+      AR AZ CO CT DC DE FL GA IA ID IL IN KY MA MD ME MI MO MS MT ND NE NJ NM PA
+      RI SC SD TX VA VT WA WI WY
+    ].freeze
+
+    SUPPORTED_STATE_ID_TYPES = %w[
+      drivers_license drivers_permit state_id_card
+    ].freeze
```

Comment:
ditto, I think this would be a good `.to_set` candidate

Reply:
yep to both